src/main/java/com/weirddev/testme/intellij/template/context/Field.java

```diff
@@ -60,7 +60,7 @@ public Field(PsiField psiField, PsiClass srcClass, TypeDictionary typeDictionary
         type= buildType(psiField.getType(), typeDictionary, maxRecursionDepth);
         this.isAnnotatedByDI = buildAnnotatedByDI(psiField, srcClass, typeDictionary);
         this.hasSetter = buildHasSetter(srcClass, psiField.getName(), typeDictionary);
-        String canonicalText = srcClass.getQualifiedName();
+        String canonicalText = srcClass.getQualifiedName() == null ? srcClass.getName() : srcClass.getQualifiedName();
```

Comment:
This will fix the NPE, but will it result in properly generated UT when there's a parent "Builder" pattern generated by Lombok ?! ideally would be good to have a TC that covers this scenario. if it's too time consuming, nevermind, will accept this change as is. Thanks!